src/cdk/activation.ts

```diff
@@ -22,9 +25,33 @@ export async function activate(activateArguments: { extensionContext: vscode.Ext
         treeDataProvider: explorer,
         showCollapseAll: true
     })
+
+    // Indicates workspace includes a CDK app and user has expanded the Node
+    view.onDidExpandElement(e => {
+        if (e.element.contextValue === 'awsCdkAppNode') {
+            ext.telemetry.record(getTelemetryEvent('appExpanded'))
+        }
+    })
+
+    // Indicates CDK explorer was disabled
+    vscode.workspace.onDidChangeConfiguration(e => {
+        const explorerEnabled = 'aws.cdk.explorer.enabled'
+        if (e.affectsConfiguration(explorerEnabled) && !vscode.workspace.getConfiguration().get(explorerEnabled)) {
```

Comment:
If we also track `explorerEnabled` , this is good information around how often the feature is being revisited.

Reply:
good call, done